### PR TITLE
Fixing Errors in regarding compositions

### PR DESCRIPTION
**Pull request by susw12**
This is an update to https://github.com/umbraco/UmbracoDocs/pull/1020.
 This address the issue https://github.com/umbraco/UmbracoDocs/issues/1012.

### CODE REVIEW

File: Getting-Started/Data/Defining-content/index.md [hunk 1]
```diff
@@ -22,6 +22,12 @@ Go to the __Settings__ section in the backoffice. On the __Document Types__ node
 ![Creating a Document Type](images/Document-Type-Create.jpg)
 _You can also choose to create a Document Type without a template and create folders to organize your Document Types._
 
+You can also use compositions to create a new document type. Compositions allows you to inhert properties from other tabs. When using a mixed setup, you can take advantage of nesting and use compositions by visiting the Structure tab. A checklist like this should appear:
```
Comment (marcemarc) [inline Getting-Started/Data/Defining-content/index.md]: inherit? or insert? but not inhert?

Comment (susw12) [reply Getting-Started/Data/Defining-content/index.md]: I am unsure what you mean by this. Please clarify.

Comment (jmayntzhusen) [reply Getting-Started/Data/Defining-content/index.md]:
```suggestion
You can also use compositions to create a new document type. Compositions allows you to inherit properties from other tabs. When using a mixed setup, you can take advantage of nesting and use compositions by visiting the Structure tab. A checklist like this should appear:
```

Comment (susw12) [reply Getting-Started/Data/Defining-content/index.md]: Oh ok, thanks.

**Comment by nul800sebastiaan** [comment]
Hi there @susw12,

First of all: **A big #H5YR** for contributing to Umbraco during Hacktoberfest! We are very thankful for the huge amount of PRs submitted, and all the amazing work you've been doing :pray:

If you haven't heard already we have had a record breaking amount of PR's in both the CMS and the Documentation repository. You can read all about the numbers in the [Umbraco Hacktoberfest Aftermath blog post](https://umbraco.com/blog/the-2018-hacktoberfest-aftermath/).

In the spirit of Hacktoberfest we've prepared some **exclusive Umbraco swag** for all our contributors - _including you_! 

[Receive your swag!](https://goo.gl/forms/smjQwEMDM5cUnz6x1) :point_left: Please follow this link to fill out and submit the form, **before November 21st 2018**.

Following this date we'll be sending out all the swag, which also means that it might not reach your doorstep before December, so please bear with us and be patient :pray:

The only thing left to say is **thank you so much for participating in Hacktoberfest**! We really appreciate the help!

Kind regards,
The Umbraco PR Team & The Documentation Curators

**Comment by susw12** [comment]
Dear @nul800sebastiaan, @dampee, @jmayntzhusen, and @marcemarc,

I would like to request my personal email to be removed from the commit message. If this could be done ASAP, that would be very much appreciated! If you could change the email to susw12@noreply.github.com that would be great!

Thanks,
@susw12

**Pull request merged**